### DISCUSSION TITLE
文档阶段性更新通知

### DISCUSSION THREAD

**Discussion by SylarLong** [SylarLong/iztro#164] [Announcements]
非常感谢大家对文档进度的关心，至此文档扫盲部分已经将基础、星盘、宫位、星耀以及四化更新完毕。

- [x] 基础扫盲
- [x] 星盘
- [x] 宫位
- [x] 星耀
  - [x] 主星
  - [x] 辅星
  - [x] 杂耀
  - [x] 神煞 
- [x] 四化
- [x] 飞星
- [ ] 格局
- [ ] 运限

文档地址：https://ziwei.pro

**Comment by s936505608** [comment]
牛逼哥!

**Comment by Saulwonderfulman** [comment]
看完了,想知道这些资料是从哪里获取的,想去看看

**Comment by garyddhh** [comment]
给作者点赞！
